x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -151,7 +156,10 @@ const GraphOverlayComponent = ({
   }, [signalIndexName, siemDefaultIndices]);
 
   return (
-    <OverlayContainer>
+    <OverlayContainer
+      data-test-subj="overlayContainer"
+      $restrictWidth={isEventViewer && fullScreen}
```

Comment:
Thanks for the fix. Would you explain the meaning behind using $ in the property name?

Reply:
It's a feature of `styled-components` called `transient props`, which can be used to [extend a component without passing props to the underlying DOM element](https://stackoverflow.com/a/62604142/12032711)

> Transient props are a new pattern to pass props that are explicitly consumed only by styled components and are not meant to be passed down to deeper component layers.

Reply:
Really cool